lib/cartopy/feature/nightshade.py

```diff
@@ -14,7 +14,7 @@
 
 class Nightshade(ShapelyFeature):
     def __init__(self, date=None, delta=0.1, refraction=-0.83,
-                 color="k", alpha=0.5, **kwargs):
+                 color="k", alpha=0.5, sub_solar_coords=None, **kwargs):
```

Comment:
`sub_solar_point`? `coords` seems like it could be an array of coordinates to me.

You will need to add documentation about what this is (tuple of latitude, longitude), the units (degrees), and how it interacts with date (takes precedence if it is provided).